package.json

```diff
@@ -0,0 +1,29 @@
+{
+  "name": "restaurant-page",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "dependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb-base": "^14.1.0",
+    "eslint-plugin-import": "^2.20.2"
+  },
+  "devDependencies": {
+    "webpack": "^5.11.1",
+    "webpack-cli": "^4.3.1"
+  },
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
We need to add some scripts.

kindly double-check on this 👇 
![image](https://user-images.githubusercontent.com/31897434/103968003-6c913580-5163-11eb-9821-3a07ffd178e7.png)

We should be able to run common commands like `build`, `development` should be in `package.json` inside \`scripts'.  [Pathwright](https://microverse.pathwright.com/library/javascript-module-v10-copy/166786/path/step/97635163/) in our own case here, we are missing it so I can't run `npm run build ` to generate `dist folder`, I should be able to run `npm run build`.
kindly add these two scripts `build` and `development`